src/components/navigation-search/navigation-search.tsx

```diff
@@ -52,7 +52,7 @@ class NavigationSearchComponent extends Component<SearchProps, SearchState> {
     const {kitchenSinkActive, toggledWithEnter} = this.props;
     if (!previousProps.kitchenSinkActive && kitchenSinkActive) {
       this._focusedByMouse = !toggledWithEnter;
-      this._inputRef?.focus();
+        setTimeout(() => this._inputRef?.focus(), 1000);
```

Comment:
`focus()` trigger scroll event and breaks transition animation.
try prevent scroll event on focus:
`this._inputRef?.focus({ preventScroll: true })`